config/sidebar-reference.json

```diff
@@ -62,6 +62,11 @@
             "source": "reference/api/dump.mdx",
             "label": "Dumps",
             "slug": "dump"
+        },
+        {
+            "source": "reference/api/experimental-features.mdx",
```

Comment:
A note on file naming: I believe that our pattern is to use underscores in filepaths!

For a direct comparison see the multi-search reference, the file of which is named `multi_search.mdx`

Not critical, but may be a good idea to maintain this convention